modules/vault/src/test/java/org/testcontainers/vault/VaultContainerTest.java

```diff
@@ -29,41 +35,47 @@
             "secret_three=password3",
             "secret_four=password4")
         .withInitCommand("secrets enable transit", "write -f transit/keys/my-key");
+    // }
 
     @Test
-    public void readFirstSecretPathWithCli() throws IOException, InterruptedException {
-        GenericContainer.ExecResult result = vaultContainer.execInContainer("vault", "kv", "get", "-format=json", "secret/testing1");
+    // readWithCli {
+    public void readFirstSecretPathWithCli() throws Exception {
+        GenericContainer.ExecResult result =
+            vaultContainer.execInContainer("vault", "kv", "get", "-format=json", "secret/testing1");
         final String output = result.getStdout().replaceAll("\\r?\\n", "");
-        assertThat(output, containsString("password123"));
+        assertThat(output).contains("password123");
     }
+    // }
 
     @Test
-    public void readSecondSecretPathWithCli() throws IOException, InterruptedException {
+    public void readSecondSecretPathWithCli() throws Exception {
         GenericContainer.ExecResult result = vaultContainer.execInContainer("vault", "kv", "get", "-format=json", "secret/testing2");
         final String output = result.getStdout().replaceAll("\\r?\\n", "");
         System.out.println("output = " + output);
-        assertThat(output, containsString("password1"));
-        assertThat(output, containsString("password2"));
-        assertThat(output, containsString("password3"));
-        assertThat(output, containsString("password4"));
+        assertThat(output).contains("password1");
+        assertThat(output).contains("password2");
+        assertThat(output).contains("password3");
+        assertThat(output).contains("password4");
     }
 
     @Test
-    public void readFirstSecretPathOverHttpApi() throws InterruptedException {
+    // readWithHttpApi {
```

Comment:
This comment isn't really necessary - we could have the `codeinclude` tag in the docs just say: `inside_block:readFirstSecretPathOverHttpApi`.